skyportal/utils/offset_util.py

```diff
@@ -0,0 +1,170 @@
+import datetime
+
+import numpy as np
+from astropy import units as u
+from astropy.coordinates import SkyCoord
+from astroquery.gaia import Gaia
+from astropy.time import Time
+
+
+facility_parameters = {'Keck': {"radius_degrees": 2.0/60,
+                                "mag_limit": 18.5,
+                                "mag_min": 11.0,
+                                "min_sep_arcsec": 4.0
+                                },
+                       'P200': {"radius_degrees": 2.0/60,
+                                "mag_limit": 18.0,
+                                "mag_min": 10.0,
+                                "min_sep_arcsec": 5.0
+                                },
+                       'Shane': {"radius_degrees": 2.5/60,
+                                 "mag_limit": 17.0,
+                                 "mag_min": 10.0,
+                                 "min_sep_arcsec": 5.0
+                                 }
+                       }
+
+def get_nearby_offset_stars(source_ra, source_dec, source_name,
+                            how_many=3,
+                            radius_degrees=2 / 60.,  # 2 arcmin radius
+                            mag_limit=18.0,
+                            mag_min=10.0,
+                            min_sep_arcsec=5,
+                            starlist_type='Keck',
+                            obstime_isoformat=None,
+                            use_source_pos_in_starlist=True,
+                            remaining_searches=1
```

Comment:
The logic here is a bit counter-intuitive: it looks as though `remaining_searches==1` actually translates to two searches being done.  Perhaps `remaining_searches` should also be checked at the beginning of the function, and should default to 2?

Reply:
I'll fix this logic.

Reply:
Ok, returning `queries_issued` now which is the number of times we call the Gaia catalog to get the requested number of offset stars.